addons/web/static/src/views/fields/monetary/monetary_field.js

```diff
@@ -30,7 +30,7 @@ export class MonetaryField extends Component {
         useInputField({
             getValue: () => this.formattedValue,
             refName: "numpadDecimal",
-            parse: parseMonetary,
```

Comment:
i found a better way to do it: you should add instead a getter ```js
get inputOptions() {
  return {
    getValue: ...,
    refName: ...,
    parse: ...,
  };
};

```
so it is easy for everyone to subclass  the monetary field and customize any option. we should acutally do that with every field component